orcid-commons/src/main/java/no/sikt/nva/orcid/commons/model/business/OrcidCredentials.java

```diff
@@ -1,118 +1,43 @@
 package no.sikt.nva.orcid.commons.model.business;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.Builder;
+import no.unit.nva.commons.json.JsonSerializable;
+
 import java.net.URI;
 import java.time.Instant;
 import java.util.Objects;
-import lombok.AccessLevel;
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.Data;
-import no.unit.nva.commons.json.JsonSerializable;
-import nva.commons.core.JacocoGenerated;
 
-@Data
+
 @Builder(
-    builderClassName = "OrcidCredentialsBuilder",
-    toBuilder = true,
-    builderMethodName = "builder",
-    buildMethodName = "build",
-    setterPrefix = "with"
+        builderClassName = "OrcidCredentialsBuilder",
+        toBuilder = true,
+        builderMethodName = "builder",
+        buildMethodName = "build",
+        setterPrefix = "with"
 )
-@AllArgsConstructor(access = AccessLevel.PACKAGE)
-public class OrcidCredentials implements JsonSerializable {
-
-    private static final String ORCID = "orcid";
-    private static final String ACCESS_TOKEN = "accessToken";
-    private static final String TOKEN_TYPE = "tokenType";
-    private static final String EXPIRES_IN = "expiresIn";
-    private static final String TOKEN_VERSION = "tokenVersion";
-    private static final String PERSISTENT = "persistent";
-    private static final String ID_TOKEN = "idToken";
-    private static final String TOKEN_ID = "tokenId";
-    private static final String MODIFIED = "modified";
-    private static final String CREATED = "created";
-
-    @JsonProperty(ORCID)
-    private URI orcid;
-
-    @JsonProperty(ACCESS_TOKEN)
-    private String accessToken;
-
-    @JsonProperty(TOKEN_TYPE)
-    private String tokenType;
-
-    @JsonProperty(EXPIRES_IN)
-    private int expiresIn;
-
-    @JsonProperty(TOKEN_VERSION)
-    private String tokenVersion;
-
-    @JsonProperty(PERSISTENT)
-    private boolean persistent;
-
-    @JsonProperty(ID_TOKEN)
-    private String idToken;
-
-    @JsonProperty(TOKEN_ID)
-    private int tokenId;
-
-    @JsonProperty(MODIFIED)
-    private Instant modified;
-
-    @JsonProperty(CREATED)
-    private Instant created;
-
-    @JacocoGenerated
-    public OrcidCredentials() {
-    }
-
-    public OrcidCredentials copy() {
```

Comment:
The copy method is no longer needed. We can't modify a record in any way, so no need for deep copy before we modify.